config/_default/menus/main.en.yaml

```diff
@@ -2769,21 +2769,26 @@ menu:
       parent: custom_instrumentation
       identifier: custom_instrumentation_dotnet
       weight: 2028
+    - name: iOS
```

Comment:
Just to confirm, this is distinct from this page? https://docs.datadoghq.com/tracing/trace_collection/custom_instrumentation/swift/

Reply:
Yes, this talks about the Otel community Swift SDK.

https://github.com/open-telemetry/opentelemetry-swift


This documentation is for https://github.com/DataDog/dd-sdk-ios